Pages/Index.cshtml

```diff
@@ -47,4 +52,6 @@
         }
 
     </div>
-</div>
\ No newline at end of file
+</div>
+</body>
+</html>
```

Comment:
Each time you have a nested tag, indent by one tab.  That makes the HTML more readable.